packages/client/foreman_ygg_migration/foreman_ygg_migration.spec

```diff
@@ -0,0 +1,34 @@
+Name: foreman_ygg_migration
```

Comment:
I am debating with myself whether this needs to be an own package, vs a sub-package of `foreman_ygg_worker` (smth like `foreman_ygg_worker_migration` or so).